lib/web_ui/test/text_editing_test.dart

```diff
@@ -2565,10 +2565,9 @@ Future<void> testMain() async {
       expect(input.style.backgroundColor, 'transparent');
       expect(input.style.caretColor, 'transparent');
       expect(input.style.outline, 'none');
-      expect(input.style.border, 'none');
+      expect(input.style.border.split(' '), contains('none'));
```

Comment:
What's the actual value? I can't tell why the split is necessary. Maybe leave a comment with an explanation?